data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -173,15 +173,15 @@
             # wmllint: local spelling weaponsmiths
             message= _ "Aye, it’s true we do prefer our axes and hammers, but then again, we make swords. We couldna’ claim to be expert weaponsmiths wi’out kenning how to wield the weapons we make."
         [/message]
-
+        
         [message]
             speaker=Hamel
             message= _ "There are some dwarves out there that are bonny fighters wi’ a sword as any ye are like to meet. Belike with bows, maces and any other weapons we fashion. In fact, Galim here is quite skilled in the art."
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "That’s great! What is the price of your weapons? We have gathered much booty from Malifor’s treasury and we will be able to pay you generously."
+            message= _ "That’s great! What is the price of your weapons? We have gathered much booty from Malifor’s treasury and will be able to pay you generously."
```

Comment:
"booty" doesn't feel right to me. I think this line either should be rewritten completely, or not change at all.

Reply:
I think I mentioned that before in another issue/PR. I think @nemaara dismissed it on the basis that NR has a more laid-back tone overall. Too, it's out-of-scope for this PR. For the change specifically, I think it's okay either way.